src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,25 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getAge() < MIN_AGE) {
+            throw new RuntimeException("The user's age must be at least " + MIN_AGE + ".");
+        }
+        if (user.getPassword().length() < MIN_PASSWORD_LENGTH) {
+            throw new RuntimeException("The user's password must be at least " + MIN_PASSWORD_LENGTH
+                    + " characters.");
+        }
```

Comment:
check password after login. You won't need to check if if user is duplicate for example